dask/setup-dask.sh

```diff
@@ -1,13 +1,11 @@
 #!/bin/bash
 set -e
 
-virtualenv dask/py-dask --python=python3
```

Comment:
the virtualenv version that is being installed installs python=3.8. We dropped support for that months ago